open-media-match/src/OpenMediaMatch/storage/interface.py

```diff
@@ -61,6 +61,7 @@ class SignalTypeConfig:
 
     # Signal types that are not enabled should not be used in hashing/matching
     enabled: bool
```

Comment:
blocking; Remove this guy - if we want to retain a way to resolve the coinflip, we could add it as a method